projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -1,36 +1,52 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Main class file for Jetpack Admin pages.
+ *
+ * @package automattic/jetpack
+ */
 
 use Automattic\Jetpack\Identity_Crisis;
 use Automattic\Jetpack\Redirect;
 use Automattic\Jetpack\Status;
 
-// Shared logic between Jetpack admin pages
+/**
+ * Shared logic between Jetpack admin pages.
+ */
 abstract class Jetpack_Admin_Page {
-	// Add page specific actions given the page hook
-	abstract function add_page_actions( $hook );
+	/** Add page specific actions given the page hook.
+	 *
+	 * @param string $hook Hook of current page.
+	 */
+	abstract public function add_page_actions( $hook );
 
-	// Create a menu item for the page and returns the hook
-	abstract function get_page_hook();
+	/**
+	 * Create a menu item for the page and returns the hook.
```

Comment:
Should document that return value.
```suggestion
	 * Create a menu item for the page and returns the hook.
	 *
	 * @return string|false Return value from WordPress's `add_menu_page()` or `add_submenu_page()`.
```